trace/http.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// traceHTTP uses httptrace to log all network activity for HTTP requests.
-func traceHTTP(ctx context.Context) context.Context {
+// HTTP uses httptrace to log all network activity for HTTP requests.
+func HTTP(ctx context.Context) context.Context {
```

Comment:
This for sure relies on `logrus` any reason why this is outside of it?

Reply:
Of course it got a dependency on logrus, but it's not a hard dependency of the `Network` part, it could also be used standalone.